data/risingwave/models/30_gold/energy/electrical_soc_dummy_data.sql

```diff
@@ -0,0 +1,24 @@
+{{ config(materialized='table_with_connector') }}
```

Comment:
This doesn't have a connector. If you just had `materialized='table'` you should be able to leave out the `CREATE TABLE {{ this }} AS`